kratos/tests/sources/test_nodal_data.cpp

```diff
@@ -80,12 +80,10 @@ namespace Kratos {
 					velocity_y_sum += i_node->FastGetSolutionStepValue(VELOCITY_Y);
 				}
 				KRATOS_CHECK_DOUBLE_EQUAL(pressure_sum, i * size);
-				KRATOS_CHECK_DOUBLE_EQUAL(velocity_x_sum, i * size);
+				KRATOS_CHECK_DOUBLE_EQUAL(velocity_x_sum, 2*i * size);
 				KRATOS_CHECK_DOUBLE_EQUAL(velocity_y_sum, 0.00);
 			}
 		}
```

Comment:
expanding on my previous comment, my concern is oriented more to the dofs than to the variables themselves.

the point is that if i have a velocity-pressure problem

it is very important for me at the moment of solving the system of equations to be sure that dofs are numbered as
VELOCITY_X VELOCITY_Y VELOCITY_Z PRESSURE
or
PRESSURE VELOCITY_X VELOCITY_Y VELOCITY_Z

but with the guarantee that for example 
VELOCITY_X PRESSURE VELOCITY_Y VELOCITY_Z
or even worst
VELOCITY_Z VELOCITY_Z ....

can not happen.

this today is guaranteed as the VELOCITY_* variables are numbered consecutive one to the other (it may well happen however that VELOCITY and VELOCITY_X are not consecutive).
The point is that when two dofs are compared, this is done by comparing first the nodeId and then the key. if the key is not guaranteed to be consecutive the mechanism will eventually fail.